### ISSUE TITLE
Getting all_workers_busy error 

### ISSUE THREAD

**Issue by konalegi** [basho/riak_cs#976]
After several weeks i got error all_workers_busy in logs.

Part of logs riak-cs error.log and app.config
https://gist.github.com/konalegi/9e4d25b8650a3bf63d48

used version
- stanchion 1.5.0
- riak 1.4.10
- riak-cs 1.5.0
- Ubuntu 12.04.4 LTS x86_64

stanchion, riak and riak-cs responses to ping with pong.
-> # riak-cs ping
pong
-> # riak ping
pong
-> # stanchion ping
pong

I can provide  additional information if need (stanchion, riak logs and configs).

**Comment by kuenishi** [comment]
What's the result of `riak-cs-gc stats` ? Maybe you're experiencing #949. If so, updating to 1.5.1 will solve it.

**Comment by konalegi** [comment]
cant run riak-cs-gc stats. there is now such command :(

```
riak-cs-gc stats
Usage: riak-cs-admin gc { batch [<leeway_seconds>] | status | pause | resume | cancel |
                          set-interval <interval_seconds> | set-leeway <leeway_seconds> }
```

i run status command, and it's output.

```
-> # riak-cs-gc status
There is no garbage collection in progress
  The current garbage collection interval is: 900
  The current garbage collection leeway time is: 86400
  Last run started at: undefined
  Next run scheduled for: 20140929T163929Z
```

**Comment by kuenishi** [comment]
Woops stats was not my intention, it's "status". Fmm, so your GC is not stalling (obvious from Gist, sorry!). According to the code, that crash message comes from pool. Maybe your cluster is under heavy load and experience 503 errors, or a very long latency?

Maybe changing app.config like this works, but not for root cause:

``` erlang
              {connection_pools,
               [
                {request_pool, {512, 0} },
                {bucket_list_pool, {5, 0} }
               ]},
```

**Comment by kuenishi** [comment]
#985 sounds like root cause.

**Comment by shino** [comment]
Fixed by #986. Will be included in 1.5.2 release.
